src/coreclr/vm/i386/asmhelpers.asm

```diff
@@ -1235,6 +1236,49 @@ _GenericCLRToCOMCallStub@0 proc public
     ret
 
 _GenericCLRToCOMCallStub@0 endp
+
+_GenericComCallStub@0 proc public
+
+    ; Pop ComCallMethodDesc* pushed by prestub
+    pop         eax
+
+    ; push ebp-frame
+    push        ebp
+    mov         ebp,esp
+
+    ; save CalleeSavedRegisters
+    push        ebx
+    push        esi
+    push        edi
+
+    push        eax         ; datum
+    sub         esp, 3*4    ; next, vtable, gscookie
```

Comment:
Can we add an asm constant here?

Reply:
What would be a good constant name be? It is hard to come up with good constant names for these random chunks of space on x86 stack.

This follows the same style as the rest of the file, most of this code is copy&paste from ComCallPreStub that immediately follows with a few adjustments.

Reply:
Could we do something like introducing `GSCookieSize` and `OFFSETOF_UnmanagedToManagedFrame_Datum` constants and using `GSCookieSize + OFFSETOF_UnmanagedToManagedFrame_Datum` here (assuming I figured out what this magic number means correctly)?

Otherwise it takes quite a bit of time to figure out exactly what's going on here.